mobx/lib/src/api/annotations.dart

```diff
@@ -1,14 +1,11 @@
 /// Internal class only used for code-generation with `mobx_codegen`.
 ///
 /// During code-generation, this type is detected to identify Store class with generated toString() method
-class MakeToString {
-//  unnamed constructor to allow parameters in Store annotation and prevent useless code refactoring for mobx users
-  const MakeToString._();
+class MakeStoreConfig {
+  const MakeStoreConfig({this.makeToString = false});
+  final bool makeToString;
 }
 
-/// Declares the annotation identifier to generate toString() on observables and computeds.
-const MakeToString toString = MakeToString._();
```

Comment:
Need to create an instance for `storeConfig`:

```dart
const MakeStoreConfig storeConfig = MakeStoreConfig._();
```

Reply:
Of course but I don't see how we add the parameter part like this. Would you be ok with:
```dart
class StoreConfig extends MakeStoreConfig {
  const StoreConfig({this.makeToString = false});
  const StoreConfig._({this.makeToString = false});
  @override
  final bool makeToString;
}
```